spring-webflux/src/main/java/org/springframework/web/reactive/socket/adapter/UndertowWebSocketSession.java

```diff
@@ -61,14 +61,21 @@ protected boolean canSuspendReceiving() {
 		return true;
 	}
 
+	@Override
 	protected void suspendReceiving() {
 		getDelegate().suspendReceives();
 	}
 
+	@Override
 	protected void resumeReceiving() {
 		getDelegate().resumeReceives();
 	}
 
+	@Override
+	protected boolean isReceivesResumed() {
+		return getDelegate().isReceivesResumed();
+	}
```

Comment:
Could we call this `isSuspended`? In both places where it's called it's negated..

Reply:
done